test/rest.test.js

```diff
@@ -45,7 +45,7 @@ describe('strong-remoting-rest', function() {
       process.env.NODE_ENV = 'test';
     }
     objects = RemoteObjects.create({ json: { limit: '1kb' },
-      errorHandler: { disableStackTrace: false }});
+      strongErrorHandler: { debug: true }});
```

Comment:
Ideally, we should preserve backwards compatibility in the sense that it is still possible to configure the error handler via `errorHandler: { disableStackTrace: false }}`.

Note that the fact that we use strong-error-handler under the hood is just an implementation detail that API consumers should not need to know about.